base/sysinfo.jl

```diff
@@ -141,12 +141,22 @@ function cpu_info()
     return cpus
 end
 
+"""
+    uptime()
+
+Gets the current system uptime.
```

Comment:
Ok will update
Btw why do some of the docstrings here have the module name prefix? Is there a policy suggesting that all unexported api need to have the  module name prefix in their docstrings. I would think that not including the prefix would be recommended.